tool/clean/clean_eks/clean_eks.go

```diff
@@ -6,16 +6,18 @@ package main
 import (
 	"context"
 	"log"
-	"strings"
 	"time"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/service/eks"
+	"github.com/grafana/regexp"
 
 	"github.com/aws/amazon-cloudwatch-agent/tool/clean"
 )
 
+var nameRegex = regexp.MustCompile("^cwagent(-operator)?-eks-integ-.*")
```

Comment:
I see another cluster name being created for helm https://github.com/aws/amazon-cloudwatch-agent-operator/blob/main/integration-tests/terraform/helm/main.tf#L14C63-L14C90. We might need to include that as well.